localmedia.js

```diff
@@ -117,20 +117,12 @@ LocalMedia.prototype.stopStream = function (stream) {
         var idx = this.localStreams.indexOf(stream);
         if (idx > -1) {
             stream.getTracks().forEach(function (track) { track.stop(); });
-
-            //Half-working fix for Firefox, see: https://bugzilla.mozilla.org/show_bug.cgi?id=1208373
-            if (shouldWorkAroundFirefoxStopStream()) {
-                this._removeStream(stream);
-            }
         }
+        this._removeStream(stream);
```

Comment:
Should this be inside the `if (idx > -1)` block?

Reply:
_removeStream does its own checks. Will update for the sake of consistency with the next chunk